### DISCUSSION TITLE
WordCamp Europe 2023 - Contributor Day

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#149] [General]
Open discussions to list ideas for the Contributor Day.

**Comment by javiercasares** [comment]
Hosting Handbook:
- Take all the sections out to the Advanced Administration Handbook and reduce everything to only Team matters.

**Reply by Crixu** [reply]
As this is something that just needs some hands to sit down and compare both it would be a great thing to start with :)

**Comment by javiercasares** [comment]
[MariaDB Checks](https://wordpress.org/plugins/mariadb-health-checks/):
- Andrew will be there, so we can help the plugin with some improvements.

**Comment by javiercasares** [comment]
PHPUnit Test Runner
- Check if there are developers to improve the project
- Do it multi-PHP
- Add options to allow testing single or Multisite via .env

**Comment by javiercasares** [comment]
Advanced Administration Handbook
